<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirstConnexion</class>
 <widget class="QWidget" name="FirstConnexion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Première connexion</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>251</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>251</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Mot de passe</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Créer un compte</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
